client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -1,23 +1,224 @@
+	{
+		field: 'email_new_order',
+		title: translate( 'New order' ),
+		subtitle: translate( 'Sent when a new order is received.' ),
+	},
+	{
+		field: 'email_cancelled_order',
+		title: translate( 'Cancelled order' ),
+		subtitle: translate( 'Sent when a new order is marked \'cancelled\'.' ),
+	},
+	{
+		field: 'email_failed_order',
+		title: translate( 'Failed order' ),
+		subtitle: translate( 'Sent when a new order is marked \'failed\'.' ),
+	},
+];
+
+// id enabled
+const customerNotifications = [
+	{
+		field: 'email_customer_on_hold_order',
+		title: translate( 'Order pending payment' ),
+		subtitle: translate( 'Sent when an order is marked \'payment pending\'.' ),
+	},
+	{
+		field: 'email_customer_processing_order',
+		title: translate( 'Processing order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment processing\'.' ),
+	},
+	{
+		field: 'email_customer_completed_order',
+		title: translate( 'Completed order' ),
+		subtitle: translate( 'Sent when an order is marked \'paid in full\'.' ),
+	},
+	{
+		field: 'email_customer_refunded_order',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+	{
+		field: 'email_customer_new_account',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+];
 
 class Settings extends React.Component {
 
 	constructor( props ) {
 		super( props );
 	}
+
+	fetchSettings = ( props ) => {
+		const { siteId, fetchSettings } = props;
+		siteId && fetchSettings( siteId );
+	}
+
+	componentDidMount = () => {
+		this.fetchSettings( this.props );
+	};
+
+	componentWillReceiveProps = newProps => {
+		if ( newProps.siteId !== this.props.siteId ) {
+			this.fetchSettings( newProps );
+		}
+	};
+
+	notificationsToggle = () => {
+		undefined;
+	}
+
+	recipientsChange = () => {
+		undefined;
+	}
+
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	render() {
+		const { loading, loaded, settings } = this.props;
+		const waiting = ! loaded || loading;
+
 		return (
 			<div className="email-settings__container">
+				<div className="email-settings__origin">
+					<Card className="email-settings__origin-title">
+						Origin
+					</Card>
+					<Card className="email-settings__origin-settings">
+						{ waiting && <p className="email-settings__loading" /> }
+						{ loaded && <NotificationsOrigin
+								item={ fromName }
+								recipient={ settings.email[ fromName.field ] }
+								onChange={ this.recipientsChange }
+							/> }
+						{ waiting && <p className="email-settings__loading" /> }
+						{ loaded && <NotificationsOrigin
+								item={ fromAddress }
+								recipient={ settings.email[ fromAddress.field ] }
+								onChange={ this.recipientsChange }
+							/> }
+					</Card>
+				</div>
+				<div className="email-settings__internal-notifications">
+					<Card className="email-settings__internal-notifications-title">
```

Comment:
The title card is a separate card, and uses the `extendedheader` component. Please see https://github.com/Automattic/wp-calypso/blob/5f15f3365213253724ca0d8798c6cb8160f00bc9/client/extensions/woocommerce/app/settings/payments/payments-offline.js#L21 for example.

Reply:
```
<ExtendedHeader
    label={ translate( 'Internal notifications' ) }
    description={ translate( 'Email notifications sent to store staff.' ) }
/>
```